src/browser/modules/DBMSInfo/MetaItems.tsx

```diff
@@ -152,12 +154,14 @@ const LabelItems = ({
       <DrawerSectionBody className={wrapperStyle}>
         {labelItems}
       </DrawerSectionBody>
-      <ShowMoreOrAll
-        total={totalNumItems}
-        shown={labels.length}
-        moreStep={moreStep}
-        onMore={onMoreClick}
-      />
+      <ThemeProvider theme={dark}>
```

Comment:
At first I was thinking we'd want to pass the theme as a prop to avoid using a ThemeProvider but then we'd have to add a prop to ShowMorOrAll, so I thnk this solution works better